platform-api/che-core-api-git/src/main/java/org/eclipse/che/api/git/GitService.java

```diff
@@ -79,10 +79,7 @@
 import java.io.ByteArrayOutputStream;
 import java.io.IOException;
 import java.net.URISyntaxException;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
Do not use `*` in imports, [https://github.com/codenvy/codenvy-dev-resources/wiki/Source-file#imports](url)